model/status_board/v1/statuses_resource.model

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright (c) 2021 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Manages the collection of statuses
+resource Statuses {
+  // Retrieves the list of statuses.
+  method List {
+    in out Page Integer = 1
+    in out Size Integer = 100
+    in CreatedAfter Date
+    in CreatedBefore Date
```

Comment:
Consider using `search` instead of these custom search criteria: `search=created_at >= ... and created_at <= ...`.